dspace-server-webapp/src/test/java/org/dspace/app/rest/HealthIndicatorsIT.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.dspace.app.rest.configuration.ActuatorConfiguration.UP_WITH_ISSUES_STATUS;
+import static org.dspace.app.rest.link.search.HealthIndicatorMatcher.match;
+import static org.dspace.app.rest.link.search.HealthIndicatorMatcher.matchDatabase;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.Map;
+
+import org.dspace.app.rest.test.AbstractControllerIntegrationTest;
+import org.junit.Test;
+import org.springframework.boot.actuate.health.Status;
+
+/**
+ * Integration tests to verify the health indicators configuration.
+ *
+ * @author Luca Giamminonni (luca.giamminonni at 4science.it)
+ *
+ */
+public class HealthIndicatorsIT extends AbstractControllerIntegrationTest {
+
+    private static final String HEALTH_PATH = "/actuator/health";
```

Comment:
While it is great that we have basic tests for the `/actuator/health` endpoint, it appears we are missing tests for `/actuator/info`.  We should be verifying that `/actuator/info` is only accessible to Admins, and that it returns basic settings that are configured by default in `actuator.cfg`.